packages/blockchain-wallet-v4-frontend/src/components/StepIndicator/index.js

```diff
@@ -49,8 +50,8 @@ const Steps = styled.div`
 
 const Step = styled.span`
   font-size: 14px;
-  min-width: 70px;
-  max-width: 70px;
+  min-width: ${props => props.coinify ? '135px' : '70px'};
+  max-width: ${props => props.coinify ? '135px' : '70px'};
```

Comment:
This component shouldn't know anything about who is using it. If we want to give it a width property that's fine.